app/presenters/document_presenter.rb

```diff
@@ -43,11 +43,11 @@ def metadata
       {
         f => document.send(f)
       }
-    end.reduce({}, :merge).merge(document_type: document.format)
+    end.reduce({}, :merge).merge(document_type: document.format).reject { |k, v| v.blank? }
```

Comment:
Too late now, but our styleguide says we shouldn't chain onto `end` - [`Never chain do...end`](https://github.com/alphagov/styleguides/blob/master/ruby.md#syntax)